### ISSUE TITLE
🐧 Longest Night ↔ v23.3.1

### ISSUE THREAD

**Issue by AnummaBrooke** [itchio/itch-compatibility-watchlist#833]
:rotating_light: game [Longest Night](https://finji.itch.io/longest-night) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/b9303e6ae750a67f836c3e38fc11e85a).

:running: Any additional details can go here!

Thank you!

**Comment by fasterthanlime** [comment]
Did you exit the game or did it crash by itself?

**Comment by AnummaBrooke** [comment]
It crashes on launch without opening. There is a very brief flicker as if about to launch, then that's gone. No error message window. 

It's the same result whether I press Launch button in Itch or try to launch the app file from its directory. 

Thank you!
Brooke

**Comment by AnummaBrooke** [comment]
(I should have added this to my reply: Lost Constellation works fine. It's only Longest Night that has this problem. )
B

**Comment by AnummaBrooke** [comment]
I do first get the configuration window for resolution and input. The launch failure happens when I hit the OK button in that window.

**Comment by fasterthanlime** [comment]
> I do first get the configuration window for resolution and input. The launch failure happens when I hit the OK button in that window.

Sounds like a game/Unity bug. Intel Integrated graphics are also known problem-causers :( Have you tried picking a lower graphics setting, seeing if launches that way?

**Comment by AnummaBrooke** [comment]
I have: various combinations of lowest-res, dire-to-good graphics quality, and windowed or not windowed.

Sorry to have thrown out my details piecemeal. Here's what I've got:

* Longest Night #139411 (latest)
* Itch 23.3.1
* Ubuntu 16.04
* Intel HD Graphics 530

Problem: When launched (whether in Itch or by calling the app file in its directory) the game presents the configuration window, but then fails to begin (without error message) when I press OK, regardless of chosen configuration settings.

Other details:

* Lost Constellation by the same game maker launches fine and plays well.
* An earlier case seems to show the same problem: Longest Night ↔ v18.6.2 #125.

Thanks for working on this with me!
Brooke

**Comment by fasterthanlime** [comment]
So we've isolated that this entirely not an app bug, but I'll leave this bug open because I'm sympathetic to your case :) hopefully we can solve it.

Being a fellow Intel Graphics victim, I have a few tricks up my sleeve: see the Workaround section on this bug: https://github.com/itchio/itch/issues/835 - do any of them help?

**Comment by littlepeppermaze** [comment]
i also have that i can't even open it, it just send me here

**Comment by AnummaBrooke** [comment]
Success.

fasterthanlime, I went to the Workaround section that you linked to. Here were my steps:

1. I called the game from within its directory in Terminal, using the Workaround in the first bullet point, like so: `./Longest\ Night.x86_64 -force-opengl`
2. That opened the configuration window, and I picked my settings, and clicked OK. The game then launched correctly.
3. At first I thought I had more trouble, because parts of the graphics were cropped from the top of the display. But, I just quit and tried again with different screen-resolution settings.
4. NOTE: Quitting the game by the game-prescribed method, pressing and holding Escape, does not quit the game. I have to quit it in Terminal with Control-C.

So, this does not get the game running properly in Itch, but it does get the game working. Thank you!

Brooke
